app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,46 @@
+<%= render partial: "tasks_all_add" %>
+
+
+<div id="task-info">
+<h2>
+   <strong>
+     <%= @task.name %>
+   </strong>
+ </h2>
+ <ul>
+   <li>
+     <strong>
+       Task:
+     </strong>
+     <%= @task.name %>
+   </li>
+   <li>
+     <strong>
+       Description:
+     </strong>
+     <%= @task.description %>
+   </li>
+   <strong>
+     Due Date:
+   </strong>
+   <%= @task.due_date.to_date.to_formatted_s(:long_ordinal) %>
+   <% if @task.complete == true && @task.completion_date != nil%>
```

Comment:
The `== true` is redundant here because `if` statements are automatically checking for [truthiness](https://learn.co/lessons/truthiness-in-ruby-readme), and booleans such as `@task.complete` have "truthiness" that matches their value of `true`/`false`.

Additionally, Rails gives us a handy method for checking if something is not `nil`, and not "empty" (e.g. a string with no characters is considered empty). So we could rewrite this line as:
```erb
<% if @task.complete && @task.completion_date.present? %>
```